docs/source/quickstart-aws.rst

```diff
@@ -88,11 +88,14 @@ Start by, copying the configuration file shown below.  Using an editor, write th
     aws-region = us-east-1
 
     [cluster]
+    machine-type = m5.8xlarge
```

Comment:
`machine-type` and `num-cpus` are defaults, so I am not sure if they are needed.

Reply:
FWIW, this was done so that if defaults change, this config file's performance wouldn't vary. This is done at the expense of brevity of the config file :(